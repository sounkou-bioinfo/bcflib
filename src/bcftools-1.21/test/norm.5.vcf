##fileformat=VCFv4.2
##reference=ref.fasta
##contig=<ID=1,length=51304566>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B
1	511	.	CAGCA	CAG,CAA,CCA	.	.	.	GT	1|0	2|1
