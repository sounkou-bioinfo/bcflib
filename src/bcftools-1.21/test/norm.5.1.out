##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##reference=ref.fasta
##contig=<ID=1,length=51304566>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	A	B
1	511	.	CAGCA	CAG	.	.	.	GT	1|0	0|1
1	511	.	CAGCA	CAA	.	.	.	GT	0|0	1|0
1	511	.	CAGCA	CCA	.	.	.	GT	0|0	0|0
